napalm/base/helpers.py

```diff
@@ -255,3 +256,47 @@ def as_number(as_number_val):
         return (int(big) << 16) + int(little)
     else:
         return int(as_number_str)
+
+
+def canonical_interface(interface, device_os, change, short=False):
```

Comment:
I think this should be two different methods; one for each direction. @mirceaulinic @ktbyers thoughts?

Reply:
I'm ok either way. To clarify to M & K, one way is short to long (et -> Ethernet), the other is long to short (Ethernet -> eth ).